automated install/basic-install.sh

```diff
@@ -98,6 +98,12 @@ if command -v apt-get &> /dev/null; then
     return 0
   }
 
+  source /etc/os-release || true
+    if [[ ${PRETTY_NAME} == "Raspbian GNU/Linux 8 (jessie)" ]]; then
```

Comment:
I'm almost inclined to just leave that out of the dependencies now, anyone that is installing Raspbian Jessie would already have the package installed, we don't need to add it anymore. Then the `/etc/dhcpcd.conf` file detection would be enough to check if that file needed modification.